flask_praetorian/base.py

```diff
@@ -69,27 +70,60 @@ class Praetorian:
     for applications and designated endpoints
     """
 
-    def __init__(self, app=None, user_class=None, is_blacklisted=None):
+    def __init__(
+                self,
+                app=None,
+                user_class=None,
+                is_blacklisted=None,
+                encode_jwt_token_hook=None,
+                refresh_jwt_token_hook=None
+    ):
         self.pwd_ctx = None
         self.hash_scheme = None
         self.salt = None
 
         if app is not None and user_class is not None:
-            self.init_app(app, user_class, is_blacklisted)
+            self.init_app(
+                        app,
```

Comment:
please follow style customs, basic pep8:

```
self.init_app(
    app,
    user_class,
    ...
)
```

Also, *always* use trailing commas for multi-line argument lists

Reply:
Tried to address the pep8 issues. Think it should be good now, but let me know if there are any additional concerns. Thanks!